exchanges/huobi/huobi_wrapper.go

```diff
@@ -1460,16 +1460,21 @@ func (h *HUOBI) GetOrderHistory(ctx context.Context, req *order.GetOrdersRequest
 			}
 			for x := range resp {
 				orderDetail := order.Detail{
-					ID:             strconv.FormatInt(resp[x].ID, 10),
-					Price:          resp[x].Price,
-					Amount:         resp[x].Amount,
-					Pair:           req.Pairs[i],
-					Exchange:       h.Name,
-					ExecutedAmount: resp[x].FilledAmount,
-					Date:           time.Unix(0, resp[x].CreatedAt*int64(time.Millisecond)),
-					Status:         order.Status(resp[x].State),
-					AccountID:      strconv.FormatInt(resp[x].AccountID, 10),
-					Fee:            resp[x].FilledFees,
+					ID:                   strconv.FormatInt(resp[x].ID, 10),
+					Price:                resp[x].Price,
+					AverageExecutedPrice: resp[x].FilledCashAmount / resp[x].FilledAmount,
+					Amount:               resp[x].Amount,
+					ExecutedAmount:       resp[x].FilledAmount,
+					RemainingAmount:      resp[x].Amount - resp[x].FilledAmount,
+					Cost:                 resp[x].FilledCashAmount,
```

Comment:
Reference: https://huobiapi.github.io/docs/spot/v1/en/#search-past-orders

filled-cash-amount (The filled total in quote currency)